modules/ROOT/pages/use-tags-in-visualizer.adoc

```diff
@@ -9,10 +9,10 @@ If a tag is assigned to an application, the application will appear in every vie
 
 == Setting the Tag Through Properties
 
-You can automate adding tags to a node by using `anypoint.platform.visualizer.tags=<name>, <name>`, either in the application's properties file or at deployment time through the Runtime Manager UI or API. 
+You can automate adding tags to a node by using `anypoint.platform.visualizer.tags=<name>, <name>`, either in the application's properties file or at deployment time through the Runtime Manager UI or API.
 
 [NOTE]
-You can only use the Runtime Manager UI for Cloudhub and Runtime Fabric deployments.
+Setting the tag using the property file is not currently supported for Mule 4 runtimes. You can only use the Runtime Manager UI for Cloudhub and Runtime Fabric deployments. 
```

Comment:
this will be fixed with AVE-319 . do we need to document this ?

Reply:
Until it is fixed, I'd say yes.